command/connect/envoy/envoy_test.go

```diff
@@ -168,6 +199,7 @@ func TestGenerateConfig(t *testing.T) {
 				AdminBindPort:         "19000",
 				LocalAgentClusterName: xds.LocalAgentClusterName,
 				Token:                 "c9a52720-bf6c-4aa6-b8bc-66881a5ade95",
+				PrometheusScrapePath:  "/metrics",
```

Comment:
This is being added to "WantArgs" everywhere since it is a default value, so even when we don't pass the flag to `consul connect envoy`, we want it to be set to "/metrics"